contrib/opencensus-ext-grpc/opencensus/ext/grpc/server_interceptor.py

```diff
@@ -28,6 +28,12 @@
 ATTRIBUTE_COMPONENT = 'COMPONENT'
 ATTRIBUTE_ERROR_NAME = 'ERROR_NAME'
 ATTRIBUTE_ERROR_MESSAGE = 'ERROR_MESSAGE'
+ATTRIBUTE_HTTP_HOST = 'HTTP_HOST'
```

Comment:
You can use the `attributes.helper` provided in `trace` similarly to [flask[(https://github.com/census-instrumentation/opencensus-python/blob/master/contrib/opencensus-ext-flask/opencensus/ext/flask/flask_middleware.py#L36) instead.